File: scode/Winners/prob8/main.py
x=input()
l=x.split(" ")
if (len(l)!=4):
    print("NO")
else:
    a=int(l[0])
    b=int(l[1])
    c=int(l[2])
    d=int(l[3])
    p=[a,b,c,d]
    p.sort()
    print(p[0]*p[2]) 

File: scode/Winners/prob8/main.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int a,b,c,d;
    cin>>a>>b>>c>>d;
    if  (!(isdigit(a)) || (!(isdigit(b)) || (!(isdigit(c)) || (!(isdigit(d))
    {
        cout<<"NO"<<endl;
    }
    else
    {
    vector <int> p;
    p.push_back(a);
    p.push_back(b);
    p.push_back(c);
    p.push_back(d);
    sort(p.begin(),p.end());
    cout<<p[0]*p[2]<<endl;
        
    }
    
    
    return 0;
}